R/resolve-links.R

```diff
@@ -0,0 +1,112 @@
+#' Resolve Reference-Style Links
+#'
+#' Commonmark treats reference-style links as regular links, which can be a
```

Comment:
Maybe add an example or a plain explanation of what reference-style links are? I see the second item in https://github.com/ropensci/tinkr#general-principles-and-solution tries to do that but I wasn't extremely clear at the time.

Reply:
Good point! I realized there is a good resource for this, actually (and a good source of tests): https://www.markdownguide.org/basic-syntax/#reference-style-links

Reply:
I have vastly expanded the documentation in fb98ee1